bundler/spec/bundler/source_list_spec.rb

```diff
@@ -441,4 +441,19 @@
       source_list.remote!
     end
   end
+
+  describe "implicit_global_source?" do
+    context "when a global rubygem source provided" do
+      it "returns a truthy value" do
+        source_list.add_global_rubygems_remote("https://rubygems.org")
+
+        expect(source_list.implicit_global_source?).to be_truthy
```

Comment:
As mentioned in other review comments, the expectations here are also reversed. This case should be falsey and the one below truthy.